### PR TITLE
Add SR-IOV NIC partitioning [webscale]

### PR DESCRIPTION
**Pull request by jboxman**
Depends on: https://github.com/openshift/openshift-docs/pull/20717

With a proper rebase, will resolve down to a single change set in one assembly.

### CODE REVIEW

**Comment by jboxman** [comment]
@zshi-redhat, did this feature make it into 4.3._z_? What is the value of _z_? What other modules and content blocks are specific to this initiative?

Thanks!

**Comment by zshi-redhat** [comment]
> @zshi-redhat, did this feature make it into 4.3._z_? What is the value of _z_? What other modules and content blocks are specific to this initiative?
> 
> Thanks!

Yes, the SR-IOV partitioning feature is in 4.3.4 and afterwards. It's part of `configuring-sriov-device.adoc ` doc. It is the only feature required for webscale from SR-IOV side.

Another related doc is how CNV should configure SR-IOV policy in order to use VF inside KubeVirt VM, but I think this is not part of OpenShift doc, rather in CNV doc. (https://issues.redhat.com/browse/CNV-2160)

**Comment by jboxman** [comment]
@zshi-redhat, is there any associated Jira with this change?

Thanks!

**Comment by zshi-redhat** [comment]
> @zshi-redhat, is there any associated Jira with this change?
> 
> Thanks!

@jboxman yes, here is the jira link: https://issues.redhat.com/browse/SDN-713

**Comment by jboxman** [comment]
@ahardin-rh, is there a process to get this merged into _webscale_? Thanks!

File: networking/hardware_networks/configuring-sriov-device.adoc [hunk 1]
```diff
@@ -9,7 +9,8 @@ You can configure a Single Root I/O Virtualization (SR-IOV) device in your clust
 
 include::modules/nw-sriov-device-discovery.adoc[leveloffset=+1]
 include::modules/nw-sriov-configuring-device.adoc[leveloffset=+1]
+include::modules/nw-sriov-nic-partitioning.adoc[leveloffset=+1]
```
Comment (ahardin-rh) [inline networking/hardware_networks/configuring-sriov-device.adoc:12]: @jboxman To add it to webscale only, you'll just need to use these conditionals around the include:
ifdef::openshift-webscale[]
include::modules/nw-sriov-nic-partitioning.adoc[leveloffset=+1]
endif::openshift-webscale[]

Thanks for getting this added in!

**Pull request merged**